Update fork

southalln opened the pull request:
Whenever it turns out to be a good time, can we update this fork from the active dev github repo at biothings?

polyg314 committed:
is it the lenght?

polyg314 committed:
um

polyg314 committed:
out there

polyg314 committed:
json

polyg314 committed:
backup res body

polyg314 committed:
json

polyg314 committed:
mmmm

polyg314 committed:
mmmm

polyg314 committed:
what is the issue

polyg314 committed:
idk

polyg314 committed:
comments

polyg314 committed:
res

polyg314 committed:
no header

polyg314 committed:
pft

polyg314 committed:
ok henny

polyg314 committed:
counter

polyg314 committed:
um

polyg314 committed:
last for the night

polyg314 committed:
back on that troubleshooting

polyg314 committed:
was not a list

polyg314 committed:
more about that list

polyg314 committed:
mas

polyg314 committed:
dont print so much, increase interval

polyg314 committed:
interval increase to 200

polyg314 committed:
less printing

polyg314 committed:
remove more prints etc.

polyg314 committed:
commented and cleaned up code to explain sync calls for mychem.info

polyg314 committed:
Delete .DS_Store

polyg314 committed:
Delete .DS_Store

polyg314 committed:
Delete .DS_Store

polyg314 committed:
more testing

polyg314 committed:
Merge branch 'master' of https://github.com/polyg314/biothings_explorer

polyg314 committed:
testing true

polyg314 committed:
KeyError not NameError

polyg314 committed:
KeyError message

polyg314 committed:
Just do an if statement

polyg314 committed:
try interval 500

polyg314 committed:
removed trailing white space

polyg314 committed:
add if only has 200 status code

polyg314 committed:
testing for use case

polyg314 committed:
more testing

polyg314 committed:
path_value_template

polyg314 committed:
commented out conditional

kevinxin90 committed:
Merge pull request #107 from polyg314/master

Synchronous API Calls for MyChem.info post requests

kevinxin90 committed:
Merge branch 'master' of https://github.com/kevinxin90/bte_schema

kevinxin90 committed:
add jupyter notebook for covid use case

kevinxin90 committed:
update jupyter notebook for graphql demo

kevinxin90 committed:
update urls for automat apis

kevinxin90 committed:
add synonym and mesh id for disease id resolution

kevinxin90 committed:
fix existing hint test failure and add new test for synonym resolution

kevinxin90 committed:
fix cord cell test error

kevinxin90 committed:
fix cord disease test error

kevinxin90 committed:
fix test for biolink and cord

kevinxin90 committed:
fix ctd test

kevinxin90 committed:
fix test for cord molecular activity api

kevinxin90 committed:
update chemical id mapping

kevinxin90 committed:
update mapping file

kevinxin90 committed:
change Reactome to REACT following biolink definitioin

kevinxin90 committed:
update config file to include more query options for hint module

kevinxin90 committed:
add test for hint module

kevinxin90 committed:
update config

kevinxin90 committed:
stringify each item

kevinxin90 committed:
update test cases

kevinxin90 committed:
remove test for hgvs ids

kevinxin90 committed:
update input id for user query dispatcher test

kevinxin90 committed:
disable test for reasonerstdapi since it changes too often

kevinxin90 committed:
fix semmed gene api test

kevinxin90 committed:
set direct=true for all biolink operations

kevinxin90 committed:
add test to check if direct=true return valid response

kevinxin90 committed:
fix typo in spec

kevinxin90 committed:
add mp as an input for phenotype2pathway association

kevinxin90 committed:
add test for biolink phenotype2pathway

kevinxin90 committed:
update reasoner api syntax

kevinxin90 committed:
update mydisease field based on recent mydisease change

kevinxin90 committed:
flake8 fix

kevinxin90 committed:
fix disease name field

kevinxin90 committed:
fix broken test

kevinxin90 committed:
update mydisease spec to include orphanet id as an input

kevinxin90 committed:
update config

kevinxin90 committed:
add frenquency as part of result

kevinxin90 committed:
add always prefix list

kevinxin90 committed:
fix semmed phenotype url

kevinxin90 committed:
add smartapi kg module

kevinxin90 committed:
add resolving ids module

kevinxin90 committed:
add call api module

kevinxin90 committed:
add predict query module

kevinxin90 committed:
update local specs

kevinxin90 committed:
update dependency

kevinxin90 committed:
update call apis module

kevinxin90 committed:
update config

kevinxin90 committed:
update query module;

kevinxin90 committed:
update id resolve module;

kevinxin90 committed:
update smartapi kg module

kevinxin90 committed:
add notebook for multiomics

kevinxin90 committed:
update dependency

kevinxin90 committed:
update smartapi local file

kevinxin90 committed:
update notebook for multiomics

kevinxin90 committed:
update call apis module

kevinxin90 committed:
update config

kevinxin90 committed:
update predict module;

kevinxin90 committed:
update local smartapi specs;

kevinxin90 committed:
add filter

kevinxin90 committed:
add visualize function

kevinxin90 committed:
update query utils

kevinxin90 committed:
update multiomics + service notebook

kevinxin90 committed:
update call api module

kevinxin90 committed:
add new filters

kevinxin90 committed:
add printer function

kevinxin90 committed:
add expand module

kevinxin90 committed:
add data folder

kevinxin90 committed:
update smartapi kg

kevinxin90 committed:
update query module

kevinxin90 committed:
update id resolving module

kevinxin90 committed:
update install instructions

kevinxin90 committed:
add node color feature

kevinxin90 committed:
update filter functions

kevinxin90 committed:
update local smartapi file

kevinxin90 committed:
update call apis module

kevinxin90 committed:
update query module

kevinxin90 committed:
add new notebook

kevinxin90 committed:
update call apis module

kevinxin90 committed:
update query module

kevinxin90 committed:
update config

kevinxin90 committed:
update local smartapi

kevinxin90 committed:
update id resolution module

kevinxin90 committed:
update colab url

kevinxin90 committed:
update smartapi

kevinxin90 committed:
update call apis

kevinxin90 committed:
update call apis module

kevinxin90 committed:
update config

kevinxin90 committed:
update smartapi

kevinxin90 committed:
add metakg notebook

kevinxin90 committed:
Merge pull request #120 from biothings/call-apis-new

Add new API Call feature

kevinxin90 committed:
add notebook for id resolver

kevinxin90 committed:
fix issue for semantic type can not be id resolved

kevinxin90 committed:
Merge pull request #121 from biothings/call-apis-new

Fix ID Resolver issue

kevinxin90 committed:
update local specs

kevinxin90 committed:
update call apis module

kevinxin90 committed:
add filter for edges outs

kevinxin90 committed:
add demo for edges out

kevinxin90 committed:
Merge pull request #122 from biothings/call-apis-new

Add edges out annotation feature

kevinxin90 committed:
fix hint error when API response is an empty list

kevinxin90 committed:
update local specs

kevinxin90 committed:
Merge pull request #123 from biothings/call-apis-new

Fix Hint Error

kevinxin90 committed:
update edges out notebook

kevinxin90 committed:
Merge pull request #124 from biothings/call-apis-new

update edges out notebook

kevinxin90 committed:
update local specs

kevinxin90 committed:
update local specs

kevinxin90 committed:
update semmed phenotype url

kevinxin90 committed:
Merge pull request #126 from biothings/call-apis-new

Update the url for semmed phenotype api

kevinxin90 committed:
fix verbose message for counting intermediate nodes

kevinxin90 committed:
Merge pull request #127 from biothings/call-apis-new

fix verbose message for counting intermediate nodes

kevinxin90 committed:
add covid19 pathway notebook

kevinxin90 committed:
update requirements file

kevinxin90 committed:
Merge pull request #130 from biothings/call-apis-new

Update requirements.txt to remove duplicate dependencies.

kevinxin90 committed:
Create syncWithNCATSTranslatorForkedRepo.yml

kevinxin90 committed:
remove ipython version

kevinxin90 commented:
> Whenever it turns out to be a good time, can we update this fork from the active dev github repo at biothings?

Sure! I'm going to set up a Github Action workflow to automatically sync between these two repos.

kevinxin90 committed:
add unittest for id resolver module

kevinxin90 committed:
better oop design for id resolver module

kevinxin90 committed:
remove python3.5 test

kevinxin90 committed:
remove data tools

kevinxin90 committed:
add github actions

kevinxin90 committed:
comment output api response section

kevinxin90 committed:
remove linting

kevinxin90 committed:
add utils

kevinxin90 committed:
Merge pull request #149 from biothings/fix_issue_147

Fix issue 147

kevinxin90 committed:
add validator class

kevinxin90 committed:
add exception module

kevinxin90 committed:
add test for validator

kevinxin90 committed:
add test for validator

kevinxin90 committed:
remove test_apis from ci

kevinxin90 committed:
fix test for hint module

kevinxin90 committed:
fix test for user query dispathcer

kevinxin90 committed:
Merge pull request #153 from biothings/resolve_ids_refactor

Resolve ids refactor

kevinxin90 committed:
update github action

kevinxin90 committed:
disable coverage

kevinxin90 committed:
Merge pull request #154 from biothings/fix_issue_147

Fix issue 147

kevinxin90 committed:
add release github actions

kevinxin90 committed:
Create python-publish.yml

kevinxin90 committed:
update version

kevinxin90 committed:
add codacy

kevinxin90 committed:
include branch name in push branches

kevinxin90 committed:
update github actions

kevinxin90 committed:
test pwd

kevinxin90 committed:
update export path

kevinxin90 committed:
try copy coverage file to /github/workspace

kevinxin90 committed:
try put coverage generation and move file in the same line

kevinxin90 committed:
another attempt

kevinxin90 committed:
fix no such file/direct error

kevinxin90 committed:
another attempt

kevinxin90 committed:
try artifact

kevinxin90 committed:
apply test to all branches

kevinxin90 committed:
update name of github action jobs

kevinxin90 committed:
fix invalid job name

kevinxin90 committed:
remove travis

kevinxin90 committed:
Merge pull request #157 from biothings/integrate-codacy-to-github-actions

Integrate codacy to GitHub actions

kevinxin90 committed:
Change test status badge from travis to GA

kevinxin90 committed:
add github actions fortesting APIs integrated in bte

kevinxin90 committed:
add test for individual apis

kevinxin90 committed:
Merge pull request #158 from biothings/fix_issue_155

Fix issue 155

andrewsu committed:
trying hacky fix

andrewsu committed:
bug fix using nest_asyncio

kevinxin90 committed:
docs: :memo: point package description to README

https://github.com/biothings/biothings_explorer/issues/166

kevinxin90 committed:
Merge branch 'master' of https://github.com/kevinxin90/bte_schema

kevinxin90 committed:
chore: :bookmark: bump pypi package version

https://github.com/biothings/biothings_explorer/issues/166

kevinxin90 committed:
ci: :construction_worker: rename publish to pypi github actions

https://github.com/biothings/biothings_explorer/issues/148

kevinxin90 committed:
ci: :construction_worker: add test before publish for github action

https://github.com/biothings/biothings_explorer/issues/148

kevinxin90 committed:
Merge pull request #167 from biothings/fix_issue_148

update github actions release process

kevinxin90 committed:
docs: :memo: add pypi badge to readme

kevinxin90 committed:
Merge pull request #168 from biothings/fix_issue_148

docs: :memo: add pypi badge to readme

andrewsu committed:
add NB for three-hop query posed at relay meeting

andrewsu committed:
Merge branch 'master' of github.com:biothings/biothings_explorer

andrewsu committed:
add TRAPI export, colab link

andrewsu committed:
adding 3-hop query on breast cancer

andrewsu committed:
minor update

kevinxin90 committed:
fix: :bug: fix smartapi kg failed to load bug

caused due to code breaking change on SmartAPI end

https://github.com/biothings/biothings_explorer/issues/171

Commit:
1.0.3

Automatically generated by python-semantic-release

kevinxin90 committed:
test: :white_check_mark: add test for metakg

kevinxin90 committed:
chore: :wrench: set up semantic version

kevinxin90 committed:
docs: :memo: update changelog

kevinxin90 committed:
refactor: :wrench: update gitignore

kevinxin90 committed:
test: :white_check_mark: add test for metakg

kevinxin90 committed:
fix: :bug: fix metakg failed to load bug

kevinxin90 committed:
Merge branch 'master' into fix_issue_171

kevinxin90 committed:
Merge pull request #172 from biothings/fix_issue_171

Fix metakg failed to load issue

kevinxin90 committed:
test: :white_check_mark: update mygene.info data integrity test

kevinxin90 committed:
ci: :construction_worker: remove syncWithNCATSRepo github action

kevinxin90 committed:
Merge pull request #174 from biothings/fix_issue_173

Fix mygene.info data integrity test

andrewsu committed:
fix asyncio

kevinxin90 committed:
feat: :sparkles: support query through TRAPI endpoint

kevinxin90 committed:
test: :white_check_mark: add test for TRAPI class

kevinxin90 committed:
test: :white_check_mark: add test for TRAPI class

kevinxin90 committed:
test: :white_check_mark: add test for TRAPI class

kevinxin90 committed:
test: :white_check_mark: fix wrong test

kevinxin90 committed:
test: :white_check_mark: fix broken test

kevinxin90 committed:
test: :white_check_mark: add test for TRAPI class

kevinxin90 committed:
test: :white_check_mark: fix broken test

kevinxin90 committed:
test: :white_check_mark: add test for TRAPI class

kevinxin90 committed:
docs: :memo: add documentation on how to use the TRAPI service

kevinxin90 committed:
fix: :bug: add response as a property for TRAPI class

kevinxin90 committed:
feat: :sparkles: allow user to select info level

kevinxin90 committed:
add jupyter notebook demo how to use bte trapi endpoint

kevinxin90 committed:
Merge pull request #178 from biothings/fix_issue_165

Allow query BTE TRAPI endpoint through Python Client

kevinxin90 committed:
docs: :memo: add documentation for how to use TRAPI endpoint

kevinxin90 committed:
docs: :memo: update changelogs

kevinxin90 committed:
Merge pull request #179 from biothings/fix_issue_165

docs: :memo: add documentation for how to use TRAPI endpoint

andrewsu committed:
update example to v1.1

andrewsu committed:
Update README.md

andrewsu committed:
Update index.rst

andrewsu committed:
archive repo
